server/models.py

```diff
@@ -404,14 +405,24 @@ class Batch(BaseModel):
         order_by="BatchResultTallySheet.created_at",
     )
 
-    # Match our session fields
-    last_edited_by_user_type = Column(String(200))
-    last_edited_by_user_key = Column(
-        String(200)
-    )  # An email for jurisdiction admins, a user ID for tally entry users
+    # Only one of the following should be populated, enforced via a check constraint defined below
     last_edited_by_support_user_email = Column(String(200))
+    last_edited_by_jurisdiction_admin_email = Column(String(200))
+    last_edited_by_tally_entry_user_id = Column(
+        String(200), ForeignKey("tally_entry_user.id", ondelete="set null")
```

Comment:
Did we want to say you can't delete a tally entry user if they've edited batches? If so, I think removing this `ondelete` will enforce that at the db level